protos/capture.proto

```diff
@@ -42,6 +49,7 @@ message FunctionCall {
   uint64 end_timestamp_ns = 5;
   int32 depth = 6;
   uint64 return_value = 7;
+  repeated uint64 registers = 8;
```

Comment:
Are we sure we shouldn't have some more precise mapping here? This way which register is which is determined only by the order and it can quickly get confusing.

Reply:
This was done so that we can decide to not send any registers and save on bandwidth. Eventually we could enable register tracking only for specific functions.

Reply:
Oh I guess we could use optional here and explicitly name the registers. But also, the register names are architecture specific... not sure if naming registers is what we want. It seems having a generic array is more flexible. We can always revisit later if we really have a need, but I'd keep this implementation for now. I have an another PR coming up that makes use of those registers, we can discuss further there if needed.

Reply:
This is proto3, all fields are optional.

As for registers being architecture specific, I don't think we should worry about that (yet, at least).

We can leave it like this for now and discuss it later on, but I think there should be some more explicit mapping.